src/squeue.rs

```diff
@@ -266,4 +267,10 @@ impl Entry {
         self.0.user_data = user_data;
         self
     }
+
+    /// Set the [`Personality`] of this event.
+    pub fn personality(mut self, personality: Personality) -> Entry {
```

Comment:
for function that have not been tested, I think it is placed behind unstable feature gate.

Reply:
Thank you, that makes sense (226bef3)